docker/README

```diff
@@ -0,0 +1,19 @@
+Container image for Zeek
```

Comment:
Let's also add a pointer to the documentation referencing these images with help on how to pull & run them.